.github/workflows/nightly-fips.yml

```diff
@@ -0,0 +1,135 @@
+name: Nightly FIPS
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  push:
+    branches: # TODO remove
+      - fips-playground
+    
+env:
+  CI_WAIT_FOR_OK_SECONDS: 60
+  CI_MAX_ITERATIONS_THRESHOLD: 60
+  CI_CLIENT_CONCURRENT_CONNECTIONS: 1
+  CI_MAX_WAIT_FOR_POD_TIME_SECONDS: 60
+  CI_MIN_SUCCESS_THRESHOLD: 1
+  OSM_HUMAN_DEBUG_LOG: true
+
+jobs:
+  build:
```

Comment:
I'm not sure this job really adds anything if we already do it for every change, so I'd be in favor of removing it here. This was originally defined as a dependency of these jobs to populate the build cache, but that's irrelevant now that the builds are all done in containers and I'm not really sure it sped things up at all in the first place.